packages/ckeditor5-ui/tests/icon/iconview.js

```diff
@@ -189,6 +189,13 @@ describe( 'IconView', () => {
 				expect( view.element.children[ 1 ].style.fill ).to.equal( 'red' );
 			} );
 		} );
+
+		describe( 'aria-hidden', () => {
+			it( 'should be added on icon creation', () => {
```

Comment:
Could you move this test to the existing `describe` block for the constructor and align it with the style of the existing tests? Since we already have a unit test for checking the template, you can extend that test instead.